### PR TITLE
Bugfix for #28

### PR DESCRIPTION
**Pull request by patrick-nicholson**
Quick patch to enable `%%scala` magic  when running under ipykernel.

### CODE REVIEW

**Comment by patrick-nicholson** [comment]
#28

File: spylon_kernel/scala_magic.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class ScalaMagic(Magic):
     def __init__(self, kernel):
         super(ScalaMagic, self).__init__(kernel)
         self.retval = None
-        self._interp = None
+        self._interp = scala_intp
```
Comment (parente) [inline spylon_kernel/scala_magic.py]: Hm. Is this really needed? `_get_scala_interpreter` initializes this variable by calling `get_scala_interpreter()` instead of reaching in and grabbing the global directly.

Comment (patrick-nicholson) [inline spylon_kernel/scala_magic.py]: No, it's not necessary. It's really just to avoid the messaging, etc. around using the magic for the first time if an interpreter already exists.

**Comment by patrick-nicholson** [comment]
@parente This PR is just a band-aid to get `%%scala` working. I'd like to really fix it by refactoring the package a bit, but we should talk about the refactoring first. If this doesn't do enough in the interim, we can close this.

**Comment by parente** [comment]
Merged. As for refactoring, yes. The cross-calling happening between the interpreter class and the magic makes the code hard to follow and maintain for sure.
